src/CryptoAxus.Application/Dto/FavoriteNftsDto.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace CryptoAxus.Application.Dto;
+
+public class FavoriteNftsDto : BaseDto
+{
+    public string? NftId { get; set; }
+
+    public string? ArtistWalletAddress { get; set; }
+
+    public string? CollectionId { get; set; }
+
+    public FavoriteNftsDto()
+    {
+    }
+}
```

Comment:
Add another constructor that takes parameters for each property and assigns values to it.

Reply:
Resolved